starter-code/pizza.js

```diff
@@ -1 +1,51 @@
 // Write your Pizza Builder JavaScript in this file.
+
+/* PRICE */
+
+var num = parseInt($("strong"));
+
+/* Activate the button toggle to display or remove ingredients */
+
+$(".btn-pepperonni").click(function() {
+  $(".pep").toggle();
+  $("aside ul li:nth-child(1)").toggle();
+});
+
+$(".btn-mushrooms").click(function() {
+  $(".mushroom").toggle();
+  $("aside ul li:nth-child(2)").toggle();
+});
+
+$(".btn-green-peppers").click(function() {
+  $(".green-pepper").toggle();
+  $("aside ul li:nth-child(3)").toggle();
+});
+
+/* Make normal crust and tomatoe sauce by default (cause it's the best :) )*/
```

Comment:
Agreed....

Reply:
By the way, you could write a function called "resetPizza" or something like that, and put all the code you need to make the pizza default as you want. Then, you'd just call that function before everything else. It would have the same effect, but be more readable and elegant.